air-script/tests/aux_trace/aux_trace.masm

```diff
@@ -25,6 +25,26 @@ proc.get_exemptions_points
     # => [g^{-2}, g^{-1}, ...]
 end # END PROC get_exemptions_points
 
+proc.compute_integrity_constraint_divisor
```

Comment:
super minor nit: would be great to add doc comments to the function with the formula for constraint divisor. Specifically:
```
(z^trace_len - 1) / ((z - g^{trace_len-2}) * (z - g^{trace_len-1}))
```

Reply:
I'm going to open a new PR and add the comments to the procedures :)